pkg/utils/sendgrid/api.go

```diff
@@ -0,0 +1,73 @@
+package sendgrid
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/sendgrid/rest"
+	"github.com/sendgrid/sendgrid-go"
+)
+
+const (
+	Host     = "https://api.sendgrid.com"
+	Contacts = "/v3/marketing/contacts"
+	Lists    = "/v3/marketing/lists"
```

Comment:
Could we suffix these with an `EP` so it's more clear in the code that it's an endpoint or URL?